1-Beginner/dollars_to_cents/js/main.js

```diff
@@ -19,7 +19,7 @@ function DollarToCents(number)
 function foo()
 {
   let dollarAmt = document.getElementById("dollar").value;
-  document.getElementById("cents").innerHTML = DollarToCents(dollarAmt);
+  document.getElementById("centsOutput").innerHTML = DollarToCents(dollarAmt);
```

Comment:
should be `coinsOutput`, also I dont see the point of adding this inside `foo()`.  Maybe add this inside `DollarToCents` before `alert(s)`